javascript/javascript2/week1/spiritAnimalNameGenerator.js

```diff
@@ -0,0 +1,68 @@
+const body = document.querySelector("body");
+
+// create a div to write output to
+const output = document.createElement("div");
+//apply few styles
+output.style.color = "green";
+body.appendChild(output);
+
+function getSpiritAnimalName() {
+  const x = document.getElementById("name").value;
+  const spiritAnimalNames = [
+    "Alligator",
+    "Ant",
+    "Badger",
+    "Bat",
+    "Bear",
+    "Bee",
+    "Blue Jay",
+    "Bobcat",
+    "Buffalo",
+    "Bull",
+    "Butterfly",
+    "Cardinal",
+  ];
```

Comment:
This array can be declared outside the function.

Reply:
the array is used only inside function, so it is declared inside. If array is declared outside a function and is usable only inside function, then the array holds memory throughout execution. i think which is unnecessary